spec/controllers/proposals_controller_spec.rb

```diff
@@ -229,9 +229,8 @@
       token = create(:api_token, step: approval)
       approval.user.add_delegate(create(:user))
 
-      post :approve, id: proposal.id, cch: token.access_token
+      get :approve, id: proposal.id, cch: token.access_token
```

Comment:
seems like we have `get` in some parts of this spec file and `post` in others-- should they all be one or the other?